pkg/controller/nodeipam/ipam/multi_cidr_range_allocator.go

```diff
@@ -699,7 +699,7 @@ func (r *multiCIDRRangeAllocator) prioritizedCIDRs(node *v1.Node) ([]*net.IPNet,
 		if clusterCIDR.IPv4CIDRSet != nil {
 			cidr, err := r.allocateCIDR(clusterCIDR, clusterCIDR.IPv4CIDRSet)
 			if err != nil {
-				klog.V(3).Infof("unable to allocate IPv4 CIDR, trying next range: %w", err)
+				klog.V(3).InfoS("Unable to allocate IPv4 CIDR, trying next range", "err", err)
```

Comment:
Don't we use ErrorS? to log errors?

Reply:
`ErrorS` is for severe problems. When an error is just another value, logging it with "err" as key is okay.

In this case the original log call was `V(3).Infof`, i.e. not a fatal event, so `klog.V(3).InfoS` is fine.

Reply:
I agree with @pohly